corehq/apps/domain/forms.py

```diff
@@ -222,6 +223,79 @@ def save(self, user, domain):
         return True
 
 
+class IPAccessConfigForm(forms.Form):
+    """
+    Form for updating a project's IP Access Configuration
+    """
+    country_allowlist = forms.MultipleChoiceField(
+        label="Allowed Countries",
+        choices=sorted(list(COUNTRIES.items()), key=lambda x: x[0]),
+        required=False,
+    )
+
+    ip_allowlist = forms.CharField(
+        label="Allowed IPs",
+        required=False,
+        help_text='IPs that will be allowed access to your project, regardless of country of origin. '
+                  'Please configure your list to be comma and space separated, '
+                  'e.g. 192.168.0.1, 192.168.1.1, 192.168.2.1',
+    )
+
+    ip_denylist = forms.CharField(
+        label="Blacklisted IPs",
```

Comment:
why "Blacklisted"?  I think of it as either we call them whitelist and blacklist or allowlist and denylist (and I think the latter pair is more in favor).